documentation/docs/libraries/nodejs/examples.mdx

```diff
@@ -136,58 +151,62 @@ You can use the following example to generate a new database and account:
 
 IOTA is based on _Unspent Transaction Output_ model. You can find a detailed explanation in the [Developer Guide to Chrysalis](https://chrysalis.docs.iota.org/guides/dev_guide#unspent-transaction-output-utxo).
 
-## Sending tokens
-You can use the following example to send tokens using an  Account  instance to any desired  address:
+## Sending Tokens
+
+You can use the following example to send tokens using an Account instance to any desired  address:
 
 <CodeBlock className="language-javascript">
   {send}
 </CodeBlock>
 
 The full function signature is `Account.send(address, amount, [options])`.
-You can use the default options. However, you can provide additional options, such as `remainderValueStrategy` which has the following strategies:
-*`changeAddress()` : Send the remainder value to an internal address
-*`reuseAddress()` : Send the remainder value back to its original address
+You can use the default options. However, you can provide additional options, such as `remainderValueStrategy` , which has the following strategies:
+
+*`changeAddress()` : Send the remainder value to an internal address.
+*`reuseAddress()` : Send the remainder value back to its original address.
 
-The `Account.send()` function returns a _wallet message_ that fully describes the given transaction. You can use the _messageId_ to check confirmation status. You can retrieve individual messages related to any given account using the `Account.listMessages()` function.
+The `Account.send()` function returns a _wallet message_ that fully describes the given transaction. You can use the _messageId_ to check confirmation statuses. You can also retrieve individual messages related to any given account using the `Account.listMessages()` function.
 
 ### Dust Protection
 
 The network uses a [dust protection](https://chrysalis.docs.iota.org/guides/dev_guide#dust-protection) protocol to prevent malicious actors from spamming the network while also keeping track of the unspent amount ( _UTXO_ ).
 
 :::info
-Micro-transaction below 1Mi of IOTA tokens can be sent to another address if there is already at least 1Mi on that address. 
-That's why we sent 1Mi in the last example, to comply with the dust protection.
 
-Dust protection also means you can't leave less than 1Mi on a spent address (leave a dust behind).
+Micro-transactions below 1Mi of IOTA tokens can be sent to another address if there is already at least 1Mi on that address. 
+That is why we sent 1Mi in the last example, to comply with the dust protection.
+
+Dust protection also means you cannot leave less than 1Mi on a spent address (leave a dust behind).
+
 :::
 
-## Backup a database
+## Backup a Database
```

Comment:
```suggestion
## Backing Up a Database
```
Or switching the previous examples to Send Tokens, Check Balance, etc.

Reply:
First option works 👌